web/src/clients/codemirror.ts

```diff
@@ -59,15 +59,31 @@ export class CodeMirrorClient extends FormatClient {
    */
   private currentSelection?: SelectionRange
 
+  /**
+   * If true the client will not send messages from the server
+   * back to the `editor`
+   */
+  private _writeOnly: boolean
```

Comment:
We should be consistent with use of underscores for private variables. I'd prefer to not use them, but happy to be outvoted if we are consistent. Is there an eslint for that?

Reply:
fair enough, I think there's a leadingUnderscore setting in the naming conventions rule. But happy to remove that underscore for now.

Reply:
Thanks